components/alert/test/alert-toast.test.js

```diff
@@ -14,7 +14,7 @@ describe('d2l-alert-toast', () => {
 
 	describe('close', () => {
 		it('should close when close button is clicked', async() => {
-			const el = await fixture(html`<d2l-alert-toast has-close-button open>message</d2l-alert-toast>`);
+			const el = await fixture(html`<d2l-alert-toast open>message</d2l-alert-toast>`);
```

Comment:
This attribute got inverted to `hideCloseButton` -- it's always there by default. @AllanKerr FYI